Assets/Scripts/StaminaBarSetColour.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class StaminaBarSetColour : MonoBehaviour {
+    public SpriteRenderer spriteRenderer;
+
+    public Sprite sprite1;
```

Comment:
use an array to store in all the images instead of defining 25 variables for this